examples/terraform-examples/jfrog-platform-aws-install/vpc.tf

```diff
@@ -0,0 +1,36 @@
+# This file is used to create the AWS VPC
+
+data "aws_availability_zones" "available" {
+    filter {
+        name   = "opt-in-status"
+        values = ["opt-in-not-required"]
+    }
+}
+
+module "vpc" {
+    source  = "terraform-aws-modules/vpc/aws"
+
+    name = "jfrog-vpc"
```

Comment:
We need to start using the [random_pet resource](https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/pet) and append it to all resources created to create uniqueness.

```
name = "jfrog-vpc-${random_pet.unique.id}"
```